index.html

```diff
@@ -14,119 +14,207 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <meta name="theme-color" content="#e30062" />
   </head>
-
   <body>
-    <header>
-      <nav>
-        <!-- in case of small screens, show a dropdown -->
-        <input type="checkbox" id="nav-toggle" aria-controls="navbar toggle" />
-        <label for="nav-toggle" class="burger-menu"> &#9776;</label>
+    <div class="first-screen">
+      <header>
+        <a
+          href="https://www.realdevsquad.com/"
+          target="_blank"
+          class="logo-link"
+        >
+          <img
+            id="logo"
+            src="../img\re-designed images\RDS-logo.png"
```

Comment:
mix of backward and forward slash?
Please read this: https://stackoverflow.com/questions/38428561/difference-between-forward-slash-and-backslash-in-file-path/38428899

Reply:
ok, I am correcting them!